dome9/common/resourcetype/resource_type.go

```diff
@@ -27,4 +27,6 @@ const (
 	AdmissionControlPolicy                       = "dome9_admission_control_policy"
 	Assessment                                   = "dome9_assessment"
 	ImageAssurancePolicy                         = "dome9_image_assurance_policy"
+	AwpAwsGetOnboardingData                      = "dome9_awp_aws_get_onboarding_data"
```

Comment:
remove "get". it's a data resource, not an operation.

Reply:
done